pkg/services/org/orgimpl/stats.go

```diff
@@ -0,0 +1,46 @@
+package orgimpl
+
+import (
+	"context"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	ExporterName = "grafana"
+)
+
+var (
+	// MStatTotalUserAccountsNoRole is a metric gauge for total number of
+	// user accounts with no basic role
+	MStatTotalUserAccountsNoRole prometheus.Gauge
+
+	Initialised bool = false
+)
+
+func init() {
```

Comment:
prefer prometheus.Registerer instead of direct registration.

example: `pkg/services/authn/authnimpl/metrics.go`